homeassistant/components/ozw/strings.json

```
@@ -7,7 +7,7 @@
     },
     "abort": {
       "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
-      "mqtt_required": "The MQTT integration is not set up"
+      "mqtt_required": "Set up the MQTT integration first; it's used to communicate with the OpenZWave daemon."
```

The standard is to first say the error. We shouldn't change that.
```suggestion
      "mqtt_required": "The MQTT integration is not set up. This integration is required as it's used to communicate with the OpenZWave daemon."
```